crates/db/store-sled/src/ol/schemas.rs

```diff
@@ -0,0 +1,41 @@
+use ssz::{Decode, Encode};
+use strata_db_types::traits::BlockStatus;
+use strata_identifiers::{OLBlockCommitment, OLBlockId};
+use strata_ol_chain_types_new::OLBlock;
+use typed_sled::codec::{CodecError, ValueCodec};
+
+use crate::{
+    define_table_with_default_codec, define_table_with_integer_key, define_table_without_codec,
+    impl_borsh_key_codec, impl_borsh_value_codec,
+};
+
+define_table_without_codec!(
+    /// A table to store OL Block data. Maps block commitment to Block
+    (OLBlockSchema) OLBlockCommitment => OLBlock
+);
+
+impl_borsh_key_codec!(OLBlockSchema, OLBlockCommitment);
```

Comment:
Is there a reason we are not doing `define_table_with_default_codec!`?
Also, since `OLBlockCommitment` contains an integer field, this should use non-borsh key encoding, if not already, because the integer will be encoded in little endian order when using borsh.

Reply:
I see below that `OLBlock` has a ssz based ValueCodec and thus two steps of table definition. Still might need to consider "seekable" key encoding.